src/irmin-pack/bench/pack_bench.ml

```diff
@@ -0,0 +1,33 @@
+open Lwt.Infix
+module Store = Irmin_pack.KV (Irmin.Contents.String)
```

Comment:
Would be nice to parametrise this over the backend, e.g. turn this into a library to put into an `irmin-bench` library that we could easily specialised for any backend. @pascutto has already start this work in https://github.com/pascutto/irmin/tree/irmin-bench ; that would be nice if we could combine this somehow.